packages/governance/test/unitTests/test-paramChangePositions.js

```diff
@@ -17,7 +17,10 @@ const negative = name => {
 };
 
 test('positions amount', t => {
-  const amountSpec = { parameterName: 'amount', key: 'something' };
+  const amountSpec = {
+    parameterName: ParamTypes.AMOUNT,
```

Comment:
You were right to replace names with 'pName' in a few places. This is not a type, it's a name.